docs/graphql/index.rst

```diff
@@ -15,6 +15,26 @@ querying. Types and Views from thee ``default`` module are exposed
 using their short names, whereas items from another module use the
 module name as a prefix.
 
+Here's an example of configuration that will set up GraphQL access to
+the database:
+
+.. code-block:: edgeql-repl
+
+    tutorial> CONFIGURE SYSTEM INSERT Port {
+    .........     protocol := "graphql+http",
+    .........     database := "edgedb",
+    .........     address := "127.0.0.1",
+    .........     port := 8888,
+    .........     user := "http",
+    .........     concurrency := 4,
+    ......... };
+    CONFIGURE SYSTEM
+
+This will expose GraphQL API on port 8888. Pointing your browser to
+``http://127.0.0.1:8888/explore`` will bring up a GraphiQL interface
```

Comment:
Could you please make GraphiQL a link pointing to their github page?

Reply:
OK